src/neo4clj/cypher.clj

```diff
@@ -64,18 +66,37 @@
        (when props (properties props))
        "]->" to ))
 
-(defn lookup
-  "Takes a lookup representation and returns its cypher equvalent
+(defn lookup-where
+  [{:keys [ref-id id props]}]
+  (cond
+    id (str "ID(" ref-id ") = " id)
+    props (where ref-id props)
+    :else nil))
```

Comment:
`cond` defaults to `nil` as fallback (when `:else` isn't defined). `:else nil` can be left out.

Reply:
Fixed

Reply:
Then the code isn't pushed.